readme.md

```diff
@@ -188,6 +207,7 @@ You can use the `out` option. This is fine for small projects, but for big proje
 
 The other option is to use `gulp-concat`. The ```tsc``` command sorts the files using the ```<reference>``` tags. ```gulp-typescript``` does this when you enable the ```sortOutput``` option. You can use the ```referencedFrom``` filter to only include files that are referenced from certain files.
 
+
 Source maps
 ----------
 Source maps have changed a bit in version 0.2.0. Here's an example gulpfile:
```

Comment:
Can you remove the part `Source maps have changed a bit in version 0.2.0.`? Version 0.2.0 is a long time ago...